### PR TITLE
Side-effect of changing username on codeowners is declared

### LINKED ISSUE #17099: Side-effect of changing username on codeowners is not declared
[maintainer edit]
This issue is reserved for the MLH Fellowship program.

### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/changing-your-github-username

### What part(s) of the article would you like to see updated?

This content is linked to on the change username screen of the account settings.

Some side effects are detailed, but not that this will impact `CODEOWNERS` files (at least from my understanding).

### Additional information

_No response_

---
[maintainer edit]
### Content plan
Add to this article a section for CODEOWNERS files. These files will need to be manually updated. When viewing the CODEOWNERS file from the web, a warning will be shown if the file contains any unknown users or users without write access.

### PR DESCRIPTION
**Pull request by OlayinkaAtobiloye**
### Why:

Closes https://github.com/github/docs/issues/17099

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Image of the GitHub Docs "Changing Your Username" page with recent content change reflected.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by OlayinkaAtobiloye** [comment]
Hi @cmwilson21, solved an issue from the project board, and raised this PR, could you take a look when chanced please? Thankls!

**Comment by cmwilson21** [comment]
👋 @OlayinkaAtobiloye Awesome! I'll get this triaged! ⚡

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md [hunk 1]
```diff
@@ -85,6 +85,10 @@ After changing your username, links to your previous profile page, such as `http
 
 After changing your username, the URLs to any public or secret gists will also change and previous links to these will return a 404 error. We recommend updating the links to these gists anywhere you may have shared them.
 
+## CODEOWNERS files
+
+After changing your username, CODEOWNERS files will need to be manually updated. When viewing the CODEOWNERS files from the web, a warning will show if the file contains any unknown user(s) or user(s) without write access. We recommend updating all relevant CODEOWNERS files with your new username.
```
Comment (rachaelrenk) [inline content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md:90]:
```suggestion
After changing your username, CODEOWNERS files need to be manually updated. When viewing the CODEOWNERS files on the web, a warning displays if the file contains any unknown user(s) or user(s) without write access. We recommend updating all relevant CODEOWNERS files with your new username.
```

Comment (rachaelrenk) [reply content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md:90]: What does `the web` mean in this context, or where would a user view the CODEOWNERS file(s) online? If possible, be more specific. For example, if `the web` actually means GitHub.com, then let's say that.

**Review by rachaelrenk — APPROVED** [review]
Great work adding this new section ✨  I made a few suggested edits for style and voice, and left one comment with a question for your consideration. Thanks! 🚀

**Comment by AllaithpPr** [comment]
Ss

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md (hunk 1 above)
Comment (OlayinkaAtobiloye) [reply content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md:90]: Hi @rachaelrenk, thanks for the review! The changes suggested were addressed in this [PR](https://github.com/github/docs/pull/23859) - which was created for another issue but happened to have the changes made here - and has now been merged in. So I will close this PR. Thank you!

**Comment by AllaithpPr** [comment]
A

**Pull request closed without merging**